docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,35 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing subscription password from OS native keychain
+
+This only applies if you are using a subscription capable distro like RHEL with {project}. 
+
+=== Windows
+
+On Windows, the password is stored using *Credential Manager*.
+Run the following command in Command Prompt to delete the stored password:
+
+----
+CMD> cmdkey /delete:minishift:<username>
+----
+
+=== Linux
+
+On Linux, the password is stored using *gnome-keyring*.
```

Comment:
s/On Linux/On Gnome based Linux systems/

Reply:
@anjannath The issue with Linux is there are so many variants of those, so we can not say in all Linux gnome-keyring is used e.g. kde based distributions.

Reply:
You should mention `On Linux the feature needs the secret service dbus interface, which is provided by GNOME Keyring`.

Reply:
@anjannath sounds good to me.

Reply:
@anjannath can you make it `On Linux the feature need to store the password using **D-Bus Secret Service API**, which is provided by **GNOME libsecret**.`

Reply:
Yes, made the change @LalatenduMohanty